packages/pieces/google-drive/src/lib/action/download-file.ts

```diff
@@ -0,0 +1,87 @@
+import { googleDriveAuth } from '../..';
+import { createAction, Property } from '@activepieces/pieces-framework';
+import { extension, lookup } from 'mime-types';
+
+export const downloadFile = createAction({
+  auth: googleDriveAuth,
+  name: 'download_ggdrive_file',
+  displayName: 'Download file',
+  description: 'Download a selected file from google drive file',
+  props: {
+    fileId: Property.ShortText({
+      displayName: 'File ID',
+      description: 'The ID of the file to download',
```

Comment:
In description, let's add this can come from new trigger or how to obtain this would be more useful.